documentation/previous-versions-quickstart.md

```diff
@@ -0,0 +1,531 @@
+				},
+			},
+		},
+			"",
+			"",
+		)
+	}
+}
+```
+
+## Request Retry Policy
+
+The SDK provides a baked in retry policy for failed requests with default values that can be configured.
+Each [client](https://godoc.org/github.com/Azure/go-autorest/autorest#Client) object contains the follow fields.
+- `RetryAttempts` - the number of times to retry a failed request
+- `RetryDuration` - the duration to wait between retries
+
+For async operations the follow values are also used.
+- `PollingDelay` - the duration to wait between polling requests
+- `PollingDuration` - the total time to poll an async request before timing out
+
+Please see the [documentation](https://godoc.org/github.com/Azure/go-autorest/autorest#pkg-constants) for the default values used.
+
+Changing one or more values will affect all subsequet API calls.
+
+The default policy is to call `autorest.DoRetryForStatusCodes()` from an API's `Sender` method.  Example:
+```go
+func (client OperationsClient) ListSender(req *http.Request) (*http.Response, error) {
+	sd := autorest.GetSendDecorators(req.Context(), autorest.DoRetryForStatusCodes(client.RetryAttempts, client.RetryDuration, autorest.StatusCodesForRetry...))
+	return autorest.SendWithSender(client, req, sd...)
+}
+```
+
+Details on how `autorest.DoRetryforStatusCodes()` works can be found in the [documentation](https://godoc.org/github.com/Azure/go-autorest/autorest#DoRetryForStatusCodes).
+
+The slice of `SendDecorators` used in a `Sender` method can be customized per API call by smuggling them in the context.  Here's an example.
+
+```go
+ctx := context.Background()
+autorest.WithSendDecorators(ctx, []autorest.SendDecorator{
+	autorest.DoRetryForStatusCodesWithCap(client.RetryAttempts,
+		client.RetryDuration, time.Duration(0),
+		autorest.StatusCodesForRetry...)})
+client.List(ctx)
+```
+
+This will replace the default slice of `SendDecorators` with the provided slice.
+
+The `PollingDelay` and `PollingDuration` values are used exclusively by [WaitForCompletionRef()](https://godoc.org/github.com/Azure/go-autorest/autorest/azure#Future.WaitForCompletionRef) when blocking on an async call until it completes.
+
+# Resources
+
+- SDK docs are at [godoc.org](https://godoc.org/github.com/Azure/azure-sdk-for-go/).
+- SDK samples are at [Azure-Samples/azure-sdk-for-go-samples](https://github.com/Azure-Samples/azure-sdk-for-go-samples).
+- SDK notifications are published via the [Azure update feed](https://azure.microsoft.com/updates/).
+- Azure API docs are at [docs.microsoft.com/rest/api](https://docs.microsoft.com/rest/api/).
+- General Azure docs are at [docs.microsoft.com/azure](https://docs.microsoft.com/azure).
+
+## Reporting security issues and security bugs
+
+Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC) <secure@microsoft.com>. You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we recei
```

Comment:
Seems the last sentence is not completed.

Reply:
thanks, added them back